packages/host-arch-arm64/host-arch-arm64.1/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "2.0"
+synopsis: "OCaml on AArch64 (64-bit)"
+description: """
+This package is installed if the underlying OCaml compiler is for 64-bit ARM.
+
+Precisely, this means `ocamlopt -config-var architecture` equals `arm64`.
+
+This package may be used in depends or conflicts fields of dependent packages
+to indicate either a requirement or an incompatibility with this
+architecture."""
+maintainer: "David Allsopp <david@tarides.com>"
+authors: "David Allsopp"
+license: "CC0-1.0+"
+homepage: "https://opam.ocaml.org"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+conflict-class: "ocaml-host-arch"
```

Comment:
These new packages should have `available` fields to avoid the CI attempting to install them.

Reply:
ping @dra27 this can be done later in another PR, I'll let you manage your own TODO list

Reply:
FYI this is the errors in the CI

```
extras
    arm32-ocaml-4.14
        host-arch-arm32.1 (failed: Failed: Build failed)
        host-arch-x86_64.1 (failed: Failed: Build failed)
        host-system-mingw.1 (failed: Failed: Build failed)
        host-system-other.1 (failed: Failed: Build failed)
        msys2-clang64.1 (failed: Failed: Build failed)
        ocaml-base-compiler.4.13.0 (failed: Failed: Build failed)
        ocaml-config.0 (failed: Failed: Build failed)
        ocaml-env-msvc64.1 (failed: Failed: Build failed)
        ocaml-system.4.13.0 (failed: Failed: Build failed)
        ocaml-variants.4.13.2+trunk (failed: Failed: Build failed)
        ocaml.4.14.0 (failed: Failed: Build failed)
        system-mingw.1 (failed: Failed: Build failed)
    arm32-ocaml-5.2
        host-arch-s390x.1 (failed: Failed: Build failed)
        host-arch-x86_64.1 (failed: Failed: Build failed)
        host-system-mingw.1 (failed: Failed: Build failed)
        host-system-msvc.1 (failed: Failed: Build failed)
        msys2-mingw64.1 (failed: Failed: Build failed)
        ocaml-base-compiler.4.14.1 (failed: Failed: Build failed)
        ocaml-base-compiler.5.2.0~alpha1 (failed: Failed: Build failed)
        ocaml-option-mingw.1 (failed: Failed: Build failed)
        ocaml-options-only-no-flat-float-array.1 (failed: Failed: Build failed)
        ocaml-system.5.1.1 (failed: Failed: Build failed)
        ocaml-variants.4.14.3+trunk (failed: Failed: Build failed)
        ocaml.4.13.0 (failed: Failed: Build failed)
    arm64-ocaml-4.14
        conf-g++.1.0 (failed: Failed: Build failed)
        msys2-clang64.1 (failed: Failed: Build failed)
        msys2.0.1.0 (failed: Failed: Build failed)
        ocaml-base-compiler.5.2.0 (failed: Failed: Build failed)
        ocaml-base-compiler.5.2.0~beta2 (failed: Failed: Build failed)
        ocaml.4.14.0 (failed: Failed: Build failed)
        ocaml.4.14.3 (failed: Failed: Build failed)
    arm64-ocaml-5.2
        host-arch-x86_64.1 (failed: Failed: Build failed)
        host-system-msvc.1 (failed: Failed: Build failed)
        msys2-clang64.1 (failed: Failed: Build failed)
        msys2-clangarm64.1 (failed: Failed: Build failed)
        msys2-mingw32.1 (failed: Failed: Build failed)
        system-mingw.1 (failed: Failed: Build failed)
        system-msvc.1 (failed: Failed: Build failed)
```

Reply:
What `available` field should they have? No other base packages have available fields?

Reply:
Although I can see they might temporarily want to conflict older compilers which haven't been updated to use them - I will indeed put that on the TODO list

Reply:
They should have `available` fields matching their architecture. E.g., `host-arch-s390x` should have `available: [ arch = "s390x" ]`. This will avoid the CI trying to build the package on, say, arm32.